tests/suites/test_suite_gcm.function

```diff
@@ -123,11 +124,16 @@ void gcm_encrypt_and_tag( int cipher_id, data_t * key_str,
         for( n1 = 0; n1 <= src_str->len; n1 += 1 )
         {
             mbedtls_test_set_step( n1 );
-            if( !check_multipart( &ctx, MBEDTLS_GCM_ENCRYPT,
-                                  iv_str, add_str, src_str,
-                                  dst, tag,
-                                  n1 ) )
-                goto exit;
+
+            for( n1_add = 0; n1_add <= add_str->len; n1_add += 1 )
```

Comment:
The nested loops make the GCM tests a bit slower than I'd like, and we could cut out a lot of values of (n1, n1add) without reducing coverage. But let's optimize later. Filed as https://github.com/ARMmbed/mbedtls/issues/4670.